content/en/getting_started/opentelemetry/_index.md

```diff
@@ -0,0 +1,277 @@
+
+```
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        endpoint: 0.0.0.0:4318
+...
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+    metrics:
+      receivers: [otlp]
+    logs:
+      receivers: [otlp]
+```
+
+### Configuring the Datadog Exporter
+
+The Datadog Exporter sends data collected by the OTLP receiver to the Datadog backend. In `otelcol-config.yaml`, the following configures the Datadog Exporter:
+
+```
+service:
+  pipelines:
+    metrics:
+      exporters: [datadog]
+    traces:
+      exporters: [datadog]
+    logs:
+      exporters: [datadog]
+```
+
+This code suffices to send runtime metrics, traces, and logs to Datadog. However, [*infrastructure metrics* require additional changes][4]. 
+
+### Configuring your application to send infrastructure metrics
+
+As an example, you will configure your OpenTelemetry Collector to forward Docker container metrics. 
+
+*Reminder: You must use Linux to send infrastructure metrics from the OpenTelemetry Collector to Datadog.*
+
+To collect container metrics, configure the [Docker stats receiver][5] in your Datadog Exporter: 
+
+1. Add a `docker_stats` block to the `receivers` section of `otel-config.yaml`:
+
+```
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+      http:
+        Endpoint: 0.0.0.0:4318
+  # add the following block
+  docker_stats: 
+    endpoint: unix:///var/run/docker.sock # default; if this is not the Docker socket path, update to the correct path
+    metrics:
+      container.network.io.usage.rx_packets:
+        enabled: true
+      container.network.io.usage.tx_packets:
+        enabled: true
+      container.cpu.usage.system:
+        enabled: true
+      container.memory.rss:
+        enabled: true
+      container.blockio.io_serviced_recursive:
+        enabled: true
+      container.uptime:
+        enabled: true
+      container.memory.hierarchical_memory_limit:
+        enabled: true
+```
+
+2. Update `service.pipelines.metrics.receivers` to include `docker_stats`:
+
+```
+service:
+  pipelines:
+    metrics:
+      receivers: [otlp, docker_stats] # <- update this line!
+```
+
+Your application now sends container metrics to Datadog for viewing in Datadog.
+
+### Using unified tagging to correlate observability data
+
+[Unified service tagging][6] ties observability data together in Datadog so you can navigate across metrics, traces, and logs with consistent tags.
+
+In the Calendar application's Docker Compose file, `./deploys/docker/docker-compose-otel.yml`, the following lines are already included to enable correlation between application traces and other observability data: 
```

Comment:
```suggestion
The Calendar application is already configured with unified service tagging:

1. Go to the Calendar application's Docker Compose file at `./deploys/docker/docker-compose-otel.yml`.
2. The following lines enable the correlation between application traces and other observability data: 
```

Reply:
Side note: I noticed there's mixed file naming of the YAML file extensions. If this reflects how it looks in the sample app that's fine, but if not, we should keep them consistently `.yaml` or `.yml`.

Reply:
This is how it is in the sample app... docker-compose-otel uses ".yml" and otelcol-config uses ".yaml"